src/Radio/index.tsx

```diff
@@ -1,5 +1,33 @@
-import { Radio as AntdRadio } from 'antd';
+import {
+  Radio as AntdRadio,
+  RadioGroupProps as AntdRadioGroupProps,
+  RadioProps as AntdRadioProps,
+} from 'antd';
+import { RadioButtonProps as AntdRadioButtonProps } from 'antd/es/radio/radioButton';
+import React from 'react';
+import styled from 'styled-components';
 
-export { RadioProps, RadioGroupProps } from 'antd';
+export type RadioProps = AntdRadioProps;
+export type RadioGroupProps = AntdRadioGroupProps;
+export type RadioButtonProps = AntdRadioButtonProps;
 
-export const Radio: typeof AntdRadio = AntdRadio;
+export const StyledRadio: typeof AntdRadio = styled(AntdRadio)<RadioProps>`
```

Comment:
The Styled- components should not be exported.

Reply:
Right, have to be internal